### PR TITLE
Sync method to cover some use cases

### PR DESCRIPTION
**Pull request by ankology**
Hi guys! Thanks for the project.
Today i arrived in a situation where i need to kill a created pid when the main process exit, using async method will not work, so, i made this pull request implementing it, using execaSync method (i also updated the execa version used in repo) and creating a new taskkillSync method. Please consider the pull request, thanks!

### CODE REVIEW

**Comment by sindresorhus** [comment]
Makes sense. You need to update the readme.

**Comment by ankology** [comment]
@sindresorhus Readme fixed! Also, i update the two methods to return execa result, can be useful. Makes sense for you? Thanks!

File: index.js [hunk 1]
```diff
@@ -35,5 +35,15 @@ export default async function taskkill(input, options = {}) {
 		arguments_.push(typeof element === 'number' ? '/pid' : '/im', element);
 	}
 
-	await execa('taskkill', arguments_);
+	return arguments_;
 }
+
+export function taskkillSync(input, options = {}) {
+	return execaSync('taskkill', parseArgs(input, options));
```
Comment (sindresorhus) [inline index.js]: `return` here is not correct. Same with the below.

Comment (ankology) [reply index.js]: Fixed.

File: index.js [hunk 2]
```diff
@@ -35,5 +35,15 @@ export default async function taskkill(input, options = {}) {
...
-	await execa('taskkill', arguments_);
+	return arguments_;
 }
+
+export function taskkillSync(input, options = {}) {
+	execaSync('taskkill', parseArgs(input, options));
+}
+
+export async function taskkill(input, options = {}) {
+	execa('taskkill', parseArgs(input, options));
```
Comment (sindresorhus) [inline index.js]: You have made a floating promise, which means the action would not be awaited and error could not be handled. Please add `await`.

File: readme.md [hunk 3]
```diff
@@ -11,18 +11,29 @@ $ npm install taskkill
 ## Usage
 
 ```js
-import taskkill from 'taskkill';
+import {taskkill, taskkillSync} from 'taskkill';
 
-await taskkill([4970, 4512]);
+await taskkill([pid, pid2]); //Async
+taskkillSync([pid, pid2]); //Sync
```
Comment (sindresorhus) [inline readme.md]: The comments are moot.

File: readme.md [hunk 4]
```diff
@@ -11,18 +11,29 @@ $ npm install taskkill
...
 
 See the [`taskkill` docs](https://technet.microsoft.com/en-us/library/bb491009.aspx) for more.
 
+The package export two methods <span style="color:orange;font-size:12pt;">**with same arguments usage**</span>:
+
 ### taskkill(input, options?)
 
-Returns a `Promise`.
+Default export and async execution. Returns a `Promise` that resolve or reject to
+execa [childProcessResult](https://github.com/sindresorhus/execa#childProcessResult) `object`.
```
Comment (sindresorhus) [inline readme.md]: Don't hard-wrap.

**Comment by ankology** [comment]
Fixed!

**Pull request merged**